.travis.yml

```diff
@@ -1,20 +1,44 @@
 language: ruby
 rvm:
   - 2.3.1
+
 branches:
   only:
     - master
     - develop
-script: bundle exec rspec --tag ~broken_in_ci
+
+services:
+  - mysql
+  - postgresql
+
+env:
+  - DB=sqlite
+  - DB=mysql
+  - DB=postgres
+
+before_install:
+  - if [[ $DB = 'sqlite' ]]; then echo "gem 'sqlite3'" > Gemfile.local; fi
+  - if [[ $DB = 'mysql' ]]; then echo "gem 'mysql2'" > Gemfile.local; fi
+  - if [[ $DB = 'postgres' ]]; then echo "gem 'pg'" > Gemfile.local; fi
+
 install:
   - bundle install --jobs=3 --retry=3
-  - bundle exec sqlite3 --version
```

Comment:
Could we keep this and add similar db `--version` calls, using a similar conditional pattern to the `before_install` block?  We've already run into some version specific problems in migrations, so we're going to want these sooner or later so we know what travis is running (unless we get that info some other way?).